sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -1,3 +1,24 @@
+namespace Azure.Storage
+{
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct StorageAudience : System.IEquatable<Azure.Storage.StorageAudience>
```

Comment:
Does this need a Blobs specific name like `BlobAudience` to match `BlobClientOptions`?  I'd still leave the property inside of it called `Storage` since it's the general Storage scope.

Reply:
Renamed to `BlobAudience`.